src/js/utilities/ajax.js

```diff
@@ -134,6 +135,11 @@ Crocodoc.addUtility('ajax', function (framework) {
             }
         };
         setHeaders(req, headers);
+
+        if (withCredentials === true) {
```

Comment:
Yeah, or you could just do req.withCredentials = withCredentials since latter is a boolean